source/gloperate-glkernel/source/stages/MultiFrameAggregationPipeline.cpp

```diff
@@ -22,7 +23,9 @@ MultiFrameAggregationPipeline::MultiFrameAggregationPipeline(gloperate::Environm
 , renderInterface(this)
 , multiFrameCount("multiFrameCount", this, 64)
 , m_renderFramebufferStage(cppassist::make_unique<gloperate::BasicFramebufferStage>(environment, "BasicFramebufferStage (Renderer)"))
-, m_aggregationFramebufferStage(cppassist::make_unique<gloperate::CustomFramebufferStage>(environment, "CustomFramebufferStage (Accumulation)"))
+, m_aggregationColorTextureStage(cppassist::make_unique<gloperate::TextureStage>(environment, "TextureStage (Color)"))
```

Comment:
Please do not use spaces in object names. The purpose of these names is to make the object hierarchy accessible via scripting (among other use cases), so the names must be proper identifiers. And they should be relatively short. These are names, not descriptions.

For example, we want to write something like this via scripting to access a stage:
  gloperate.canvas0.pipeline.DemoPipeline.TextureStage

That would not work well with "BasicFramebufferStage (Renderer)".